src/cache/cache_las.c

```diff
@@ -683,7 +684,7 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
          * and start timestamp WT_TS_NONE, it is simply ignored.
          */
         for (; upd != NULL; upd = upd->next) {
-            if (upd->txnid == WT_TXN_ABORTED)
+            if (upd->txnid == WT_TXN_ABORTED || F_ISSET(upd, WT_UPDATE_HISTORY_STORE))
```

Comment:
Can you remove this line of change for now? I'll handle it in a later change. It is not correct here.